Sources/Logging/Loggers/Log+DefaultLogger.swift

```diff
@@ -28,18 +28,19 @@ extension Log {
         public var destinations: [AnyMetadataLogDestination<MetadataKey>] { return _destinations.value }
 
         /// The logger's registered modules (read only).
-        public var modules: [Module : Log.Level] { return _modules.value }
+        public var modules: [Module: Log.Level] { return _modules.value }
 
         /// The logger's registered destinations. The destinations are stored as type erased versions to enable storing
         /// multiple `MetadataLogDestination`'s with the same `MetadataKey`.
         private let _destinations = Atomic<[AnyMetadataLogDestination<MetadataKey>]>([])
 
         /// The logger's registered modules.
-        private let _modules = Atomic<[Module : Log.Level]>([:])
+        private let _modules = Atomic<[Module: Log.Level]>([:])
 
         /// The logger's error callback closure, invoked whenever any of its destinations fails an operation.
-        public var onError: ((LogDestination, Error) -> ())? = { destination, error in
-            print("💥[Alicerce.Log]: Failed to perform operation in destination '\(destination.id)' with error: \(error)")
+        public var onError: ((LogDestination, Error) -> Void)? = { destination, error in
+            print("💥[Alicerce.Log]: Failed to perform operation in destination " +
+                "'\(destination.id)' with error: \(error)")
```

Comment:
Even though we don't have a rule for this, we typically break the string "as late as possible", except if breaking it sooner makes it easier to read™. 💅